.coveragerc

```diff
@@ -609,6 +609,7 @@ omit =
     homeassistant/components/switch/telnet.py
     homeassistant/components/switch/tplink.py
     homeassistant/components/switch/vesync.py
+    homeassistant/components/switcher_kis/*
```

Comment:
Once the issues are addressed more tests should be written to make this exclusion unnecessary.

Reply:
I have written tests for the component.
I think the [documentation](https://developers.home-assistant.io/docs/en/development_checklist.html) on the `.coveragerc` file is not clear:

> The .coveragerc file is updated to exclude your platform if there are no tests available or your new code uses a third-party library for communication with the device, service, or sensor

Based on "or your new code uses a third-party library for communication with the device", I assumed I should add this line in the `.coveragerc` file.

Reply:
@andrewsayre 
I'm almost done my friend, thank you for all of your inputs.
I need a final say on this issue though:
- I do have tests for the component.
- The component does use a third-party library.

Should it be included in the `.coveragerc` file?